File: Baekjoon/구현/10162.cpp
// https://www.acmicpc.net/problem/10162

#include <iostream>
using namespace std;

int main() {
    int t;
    cin >> t;

    if (t % 10 != 0) {
        cout << -1 << '\n';
    } else {
        cout << t / 300 << ' ';
        t %= 300;
        cout << t / 60 << ' ';
        t %= 60;
        cout << t / 10 << '\n';
    }

    return 0;
}

File: Baekjoon/구현/10162.py
# https://www.acmicpc.net/problem/10162

import sys
input = sys.stdin.readline

n = int(input())

if n % 10 != 0:
    print(-1)
else:
    A = B = C = 0
    A = n // 300
    B = (n % 300) // 60
    C = (n % 300) % 60 // 10
    print(A, B, C)
